lib/domain/token.js

```diff
@@ -1,10 +1,11 @@
 import ArgumentError from './argument-error';
 
 export default class Token {
-  constructor({ token, sharedDate }) {
+  constructor({ token, sharedDate, previouslySharedOn }) {
     if (!token) throw new ArgumentError('no token');
     this.token = token;
     this.sharedDate = sharedDate || null;
+    this.previouslySharedOn = previouslySharedOn || [];
```

Comment:
I'm not sure we need this? I don't know if I'd say sending an auto-reminder is the same as sharing a plan

Reply:
hm, I see, Chris W mentioned the ideally we will preserve the history of when the plan was first shared.
I agree that this is a little bit 'meh', but do we display an old date in 'last shared with resident' section then?

Reply:
an auto-reminder isn't the same as an officer sharing the plan with the resident

Reply:
so, we dont set the shared field at all when sending reminder

Reply:
and we just send the same token as last time

Reply:
👌